book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- According to the project requirements, please ensure that the `rentals` attribute has both getter and setter methods, please fix this 🙏

Reply:
@mckent05 it doesn't say that rentals should have getter and setter, it is only author and title
![image](https://user-images.githubusercontent.com/50754458/216134242-3ba71b4c-8115-4375-b9ac-7122489a2650.png)